src/stores/customer.ts

```diff
@@ -80,7 +82,7 @@ function getCustomerSubscriptionData(subscription: undefined | null | any) {
     return {
       isIncompleteSubscription: false,
       isPro: false,
-      isProPlus: false,
```

Comment:
`[Plan.PRO_PLUS]` should not be removed to ensure compatibility with legacy systems